wiki/People/Featured_Artists/Featured_Artist_playlists/en.md

```diff
@@ -210,11 +214,31 @@ As of June 2023, the above badges are awarded to any user who achieves 10 points
 ### March 2023
 
 - Top 8 beatmaps from the [Twin Trials Contest](/wiki/Contests/Twin_Trials_Contest) with separate prizing
+- All game modes
 
-### June 2023
+### May 2023
```

Comment:
wouldn't June be more correct here? considering that the playlists opened up on June 3rd according to [this newspost](https://osu.ppy.sh/home/news/2023-06-03-new-featured-artist-andora#featured-artist-playlists)

Reply:
i think it'll be better to go by the name of the playlist in lazer (which is may)